src/layerstack.stanza

```diff
@@ -236,24 +243,25 @@ public defn LayerSpec (
 
 doc: \<DOC>
 Create a Copper layer
+  Usage: Copper(0.3) : specify thickness 
+         Copper(0.3, name = "Cu") : specify thickness and name
+         Copper(0.3, AluminumMaterial, name = "Cu") : specify thickness, material and name
```

Comment:
In the third example: 

```
Copper(0.3, AluminumMaterial, name = "Cu")
```

We should probably use `name = "PWR"` instead of `Cu` which is short hand for Copper.

Reply:
This is still outstanding, I believe.